_scripts/eslint-rules/plugin.mjs

```diff
@@ -1,11 +1,11 @@
-import useI18nPolyfillRule from './use-i18n-polyfill-rule.mjs'
+import preferUseI18nPolyfillRule from './perfer-use-i18n-polyfill-rule.mjs'
```

Comment:
Looks like a typo in the filename

```suggestion
import preferUseI18nPolyfillRule from './prefer-use-i18n-polyfill-rule.mjs'
```

Reply:
Gah typos. Serves me right for doing the same thing twice and only testing it the first time (or I could just use my git stash next time).